text/0000-cbool.md

```diff
@@ -0,0 +1,28 @@
+- Feature Name: cbool
+- Start Date: Mon Mar  9 00:16:53 CET 2015
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Specify that `bool` is compatible with the `_Bool` C type.
+
+# Motivation
+
+You cannot safely call ffi functions with boolean arguments without a compatible type.
+
+# Detailed design
+
+Specify that `bool` is compatible with the `_Bool` C type and inform the compiler that `bool` is ffi safe.
+
+# Drawbacks
+
+None right now. This should be a no-op on all supported platforms.
```

Comment:
It's not ridiculous: the C `_Bool` type on some platforms may have historical constraints that don't apply to apply to Rust. In any case, the precise representation of C types is usually chosen by the compiler, not the hardware.

A minimum improvement that would help assuage my concerns would be quoting the C standard's definition of `_Bool` in the RFC.